katas/content/superdense_coding/index.md

```diff
@@ -0,0 +1,30 @@
+# Superdense Coding Kata
+
+@[section]({
+    "id": "./superdense_coding.ipynb__overview",
+    "title": "Overview"
+})
+
+**Superdense Coding** quantum kata is a series of exercises designed to get you familiar with programming in Q#.
+
+It covers the superdense coding protocol which allows us to transmit two bits of classical information by sending just one qubit using previously shared quantum entanglement.
```

Comment:
We tend to drop the "this is a series of exercises" part of the intro in the new tutorials, and jump right into the topic, especially for a kata that is that late in the series - if the learner made it so far, they probably got the idea :-)

```suggestion
Superdense coding protocol allows us to transmit two bits of classical information by sending just one qubit using previously shared quantum entanglement.
```